src/action/install_date.rs

```diff
@@ -0,0 +1,267 @@
+// Copyright 2020 Google LLC
+//
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file or at https://opensource.org/licenses/MIT.
+
+//! A handler and associated types for the install date action.
+//!
+//! The install date action uses various heuristics to detect the operating
+//! system install date.
+
+use std::time::{SystemTime, Duration};
+
+use log::error;
+use crate::session::{self, Session};
+
+#[cfg(target_family = "unix")]
+use std::{fs, path::Path};
+
+#[cfg(target_os = "windows")]
+use winreg::RegKey;
```

Comment:
It can be in the function which uses it, I think.

Reply:
Good point :) Thanks, it's fixed now.